src/applications/discharge-wizard/components/v2/resultsComponents/Warnings.jsx

```diff
@@ -61,8 +60,7 @@ const Warnings = ({ formResponses }) => {
       <AlertMessage
         content={alertContent}
         isVisible={
-          boardToSubmit.abbr === 'DRB' &&
-          RESPONSES.COURT_MARTIAL_3 === courtMartial
+          boardToSubmit.abbr === 'DRB' && RESPONSES.NOT_SURE === courtMartial
```

Comment:
Curious: is there anywhere else we use raw string comparison for `'DRB'`? Wondering if we could benefit from using a constant here.

Reply:
Nice I like that idea.